pkgs/by-name/me/mesonlsp/patches/disable-tests-that-require-network-access.patch

```diff
@@ -0,0 +1,75 @@
+From 09ff92f84ffa5b3df826ff5e42473d23a85ec393 Mon Sep 17 00:00:00 2001
+From: Pavel Sobolev <contact@paveloom.dev>
+Date: Sat, 30 Mar 2024 12:49:48 +0300
+Subject: [PATCH] Disable tests that require network access.
+
+---
+ tests/integration/meson.build |  2 +-
+ tests/libutils/test.cpp       | 38 +++++++++++++++++------------------
+ 2 files changed, 20 insertions(+), 20 deletions(-)
+
+diff --git a/tests/integration/meson.build b/tests/integration/meson.build
+index eb61ca75..f9ecb31e 100644
+--- a/tests/integration/meson.build
++++ b/tests/integration/meson.build
+@@ -47,7 +47,7 @@ wrap_files = files(
+     'wrap-test/vorbis.wrap',
+ )
+
+-test('wrap-test', wrap_tester, args: [wrap_files], timeout: 200000)
++# test('wrap-test', wrap_tester, args: [wrap_files], timeout: 200000)
+
+ partial_interpreter_tests = [
+     [
+diff --git a/tests/libutils/test.cpp b/tests/libutils/test.cpp
+index 2b20191c..ec4d6eb7 100644
+--- a/tests/libutils/test.cpp
++++ b/tests/libutils/test.cpp
+@@ -131,25 +131,25 @@ TEST(UtilsTest, testMergingDirectories) {
+   ASSERT_EQ('a', std::ifstream(outputDir / "i1/a.txt").get());
+ }
+
+-TEST(UtilsTest, testDownloadAndExtraction) {
+-  auto zipFileName = std::filesystem::path{randomFile() + "-1"};
+-  auto result = downloadFile(
+-      "https://github.com/JCWasmx86/mesonlsp/archive/refs/heads/main.zip",
+-      zipFileName);
+-  ASSERT_TRUE(result);
+-  auto directoryName = std::filesystem::path{randomFile() + "-2"};
+-  std::filesystem::create_directory(directoryName);
+-  result = extractFile(zipFileName, directoryName);
+-  ASSERT_TRUE(result);
+-  auto mustExist =
+-      directoryName / "mesonlsp-main/Benchmarks/extract_git_data.sh";
+-  ASSERT_TRUE(std::filesystem::exists(mustExist));
+-  auto mustFailFilename = std::filesystem::path{randomFile() + "-3"};
+-  result =
+-      downloadFile("lnfvwoefvnwefvwvipwnefv2efvpov2nvov", mustFailFilename);
+-  ASSERT_FALSE(result);
+-  ASSERT_FALSE(std::filesystem::exists(mustFailFilename));
+-}
++// TEST(UtilsTest, testDownloadAndExtraction) {
++//   auto zipFileName = std::filesystem::path{randomFile() + "-1"};
++//   auto result = downloadFile(
++//       "https://github.com/JCWasmx86/mesonlsp/archive/refs/heads/main.zip",
++//       zipFileName);
++//   ASSERT_TRUE(result);
++//   auto directoryName = std::filesystem::path{randomFile() + "-2"};
++//   std::filesystem::create_directory(directoryName);
++//   result = extractFile(zipFileName, directoryName);
++//   ASSERT_TRUE(result);
++//   auto mustExist =
++//       directoryName / "mesonlsp-main/Benchmarks/extract_git_data.sh";
++//   ASSERT_TRUE(std::filesystem::exists(mustExist));
++//   auto mustFailFilename = std::filesystem::path{randomFile() + "-3"};
++//   result =
++//       downloadFile("lnfvwoefvnwefvwvipwnefv2efvpov2nvov", mustFailFilename);
++//   ASSERT_FALSE(result);
++//   ASSERT_FALSE(std::filesystem::exists(mustFailFilename));
++// }
```

Comment:
```suggestion
```
lets make the patch a bit shorter and easier to handle by just deleting the lines. Make sure to generate this locally, so that the line numbers above are correct